source/collections/LinkedList.ooc

```diff
@@ -98,12 +98,11 @@ LinkedList: class <T> {
 			data = this _head prev data@
 		data
 	}
-	removeAt: func (index: SSizeT) -> T {
-		item := null
+	removeAt: func (index: Int) -> T {
+		item: T = null
```

Comment:
I don't even want to guess. Maybe it worked fine, but I've had to cast `null` sometimes in arguments, so might as well be explicit here.